frontend/src/component/ui/ViewChangeLogButton.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component, Fragment } from 'react';
+import { ChangeLogModal } from './ChangeLogModal';
+import { Update } from '../../entity/Update';
+
+interface Props {
+  onClick?: () => void;
+  changeLog?: Array<Update>;
+  closeModal: () => void;
+  shouldShowModal: boolean;
+  defaultVisibleLogs: number;
+}
+
+export class ViewChangeLogButton extends Component<Props> {
+  render() {
+    return (
+      <Fragment>
+        <div className={'row view-changelog'} onClick={this.props.onClick}>
+          <a href={'/#'}>View Changelog</a>
+        </div>
+        <ChangeLogModal {...this.props} />
```

Comment:
The responsibility of this component is to let its parent component know it's being clicked. This follows single responsibility principle: http://s/srp. In term of what the the parent components want to do, it's up to them. They can fetch data, open the modal or do something else.  Let's put this Modal in Home page component for better decoupling. This design provide our flexibility of making changes without modifying the other of system.